changelog_page.php

```diff
@@ -232,8 +229,11 @@ function print_project_header_changelog ( $p_project_name ) {
 
 			# check limit_Reporter (Issue #4770)
 			# reporters can view just issues they reported
-			if( ON === $t_limit_reporters && $t_user_access_level_is_reporter &&
-							!bug_is_user_reporter( $t_row['id'], $t_user_id )) {
+			$t_limit_reporters = config_get( 'limit_reporters' );
+			$t_report_bug_threshold = config_get( 'report_bug_threshold', null, null, $t_project_id ); // == access_get_project_level( $t_project_id ) );
+			if( ON == $t_limit_reporters
+					&& !access_has_project_level( access_threshold_min_level( $t_report_bug_threshold ) + 1, $t_project_id )
```

Comment:
This code is iterating the bugs to add to the list.
IF the option to limit reporters is set
AND we are not (project access level) GREATER THAN reporter 
AND we are not the reporter user in the bug.
THEN do not show this bug

The logic is negated, as if we were higher that a reporter, we could see all bugs